### PR TITLE
snake4: add page

### PR DESCRIPTION
**Pull request by sahilister**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by sahilister** [comment]
Navigation in the game can be done with 4 different sets of key sets. Arrow keys being the easiest has been added to this commit.

File: pages/linux/snake4.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# snake4
+
+> Snake game in terminal.
```
Comment (sbrl) [inline pages/linux/snake4.md:3]: Perhaps this will read better?
```suggestion
> Snake game in the terminal.
```

File: pages/linux/snake4.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://manpages.debian.org/snake4/snake4.6.en.html>.
+
+- Start snake game:
```
Comment (sbrl) [inline pages/linux/snake4.md:6]: Perhaps this will read better?
```suggestion
- Start a snake game:
```

File: pages/linux/snake4.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Pause game:
+
+`Spacebar`
+
+- Quit game:
+
+`q`
+
+- Show highscores:
```
Comment (sbrl) [inline pages/linux/snake4.md:26]: Perhaps this will read better?
```suggestion
- Show the high scores:
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page! I've left some comments below for you to review.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Pull request merged**